hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/WebHDFS.md

```diff
@@ -462,7 +462,7 @@ See also: [`destination`](#Destination), [FileSystem](../../api/org/apache/hadoo
 * Submit a HTTP DELETE request.
 
         curl -i -X DELETE "http://<host>:<port>/webhdfs/v1/<path>?op=DELETE
-                                      [&recursive=<true |false>]"
+                                      [&recursive=<true|false>][&skiptrash=<true|false>]"
 
```

Comment:
nit: if we can emphasize the default value of `recursive` (`false`) and `skiptrash` here in the doc it would be great! Try bold font: `[&recursive=<true|**false**>]`

Reply:
Unfortunately, making these values bold by `**` is not working as this text is covered by scrollable (Insert code mode).
However, let me make special note of default value right below the curl command.
Thanks for the suggestion.

Reply:
Done. @smengcl does this look good?

Reply:
Looks good. Thanks!

Reply:
Yes this would be good enough. Thanks!